src/index.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process')
+
+module.exports = function (pluginConfig, config, cb) {
+  exec('git tag', (err, stdout, stderr) => {
+    if (err) return cb(err)
+
+    const versionRegex = /^v((?:[0-9]+\.){2}[0-9]+)$/
+
+    const tags = stdout
+                  .trim()
```

Comment:
Here, I was aligning the chain with the thing it was chaining... That's a pretty common thing to do.

I can change it if you'd like, but it would look very weird.